README.md

```diff
@@ -1,2 +1,161 @@
-# portfolio
-My portfolio containing information about me
+<h1 id = 'top'><b>Portfolio</b></h1>
+
+[![Status badge](https://img.shields.io/badge/CONTRIBUTORS-0-red.svg)](https://shields.io/)
+[![Status badge](https://img.shields.io/badge/FORKS-0-yellow.svg?style=plastic&logo=appveyor)](https://shields.io/)
+[![Status badge](https://img.shields.io/badge/REVIEWS-3-<red>.svg?style=plastic&logo=appveyor)](https://shields.io/)
+[![Status badge](https://img.shields.io/badge/STARS-0-red.svg?style=plastic&logo=appveyor)](https://shields.io/)
+[![Status badge](https://img.shields.io/badge/ISSUES-16-red.svg?style=plastic&logo=appveyor)](https://shields.io/)
+[![Status badge](https://img.shields.io/badge/LICENSE-0-blue.svg?style=plastic&logo=appveyor)](https://shields.io/)
+
+<details id = "toc">
+<summary><b>Table of Contents</b></summary>
+
+1. [About The Project](#about)
+    - [Built with](#built)
+2. [Getting Started](#started)
+3. [Usage](#usage)
+4. [Road Map](#map)
+5. [Contributing](#contribution)
+6. [License](#license)
+7. [Contact](#contact)
+8. [Aknowledgements](#end)
+
+</details>
+
+---
+
+<h2 id = "about"><b> About The Project</b></h2>
+
+---
+
+![portfolioscreenshot](https://github.com/Kadunapikin/portfolio/assets/107532013/46316ca8-be63-4641-bf38-b6559af07ba0)<br> 
+
+<b>This is a Portfolio Project which contains the documentation of the user portfolio such as</b>
+
+- It contains a Home page which shows and introduction of the Portfolio owner
+- It has an about page which contains a brief description of and skills possessed by the owner.
+- It has a contact page which can be used to reach the owner.
+- The page is responsive for mobile screens
+- It contains a project link that shows a list of projects images and links done by the owner.
+- It is a project containing multiple sections and linked to the navigation bars by IDs.<br>
+<u>[Back to top](#top)</u>
+
+    <u>[Go To End](#end)</u>
+
+---
+<h2 id = "built"><b>Built With</b></h2>
+
+---
+![Javascript Badge](https://img.shields.io/badge/JavaScript-F7DF1E?style=for-the-badge&logo=javascript&logoColor=black) <br>![CSS Badge](https://img.shields.io/badge/CSS3-1572B6?style=for-the-badge&logo=css3&logoColor=white) <br> ![HTML5 Badge](https://img.shields.io/badge/HTML5-E34F26?style=for-the-badge&logo=html5&logoColor=white)<br>
+<u>[Go Back One Step](#about)</u>
+
+<u>[Back to top](#top)</u>
+
+---
+<h2 id = "started"><b>Getting Started</b></h2>
+
+---
+To assess this webpage, here are simple steps to follow:
+
+1. Open your browser
+2. In the search or input box, type in the portfolio web address <https://github.com/Kadunapikin/portfolio/> and you will be directed to the home page.
```

Comment:
This link doesn't open a valid webpage. Please check it.

Reply:
Link has been updated now sir